apps/studio/src/store/modules/LicenseModule.ts

```diff
@@ -53,14 +53,17 @@ export const LicenseModule: Module<State, RootState>  = {
       return state.licenses.length === 0
     },
     isUltimate(state) {
+      // return true // for some reason on macos, the switching license type doesn't work so this will do for now...
```

Comment:
remove comments? Sounds like the menu may be broken? Worth fixing?

Reply:
I was leaving that there because I've accidentally committed "return true" more often than I care to admit :D 

I wasn't sure if it was worth fixing. I've looked into it a couple times and it didn't feel a "juice worth the squeeze" cliche especially since it was only on mac as far as I could tell.